src/main/webapp/app/shared/model/firebase/firebase.model.ts

```diff
@@ -165,7 +168,10 @@ export class Mutation {
   germline_genomic_indicators: GenomicIndicator[] = [];
   mutation_effect: MutationEffect = new MutationEffect();
   mutation_effect_uuid: string = generateUuid();
+  mutation_effect_comments?: Comment[] = []; // used for somatic
+  mutation_effect_germline_comments?: Comment[] = [];
```

Comment:
I checked an the old Curation Platform pulls from this location (since I can see old comments in the new platform which reads from this location). I checked BRAF, V600 to verify

Reply:
@bprize15 the old curation platform doesn't have germline integrated.

Reply:
`mutation_effect_comments` was in this location, so I was able to put `mutation_effect_germline_comments` wherever I wanted. I figured it made the most sense to keep it in the same object as `mutation_effect_comments`

Reply:
I figured it would makes sense to leave the comments on the same level of germline which is under mutaiton_effect. We can merge as it and come back to update when needed.